packages/plugins/flyte-api/package.json

```diff
@@ -12,15 +12,20 @@
         "registry": "https://registry.npmjs.org/"
     },
     "scripts": {
+        "clean": "rm -rf dist && rm -rf lib",
         "build": "yarn build:esm && yarn build:cjs",
         "build:esm": "tsc --module esnext --outDir lib/esm --project ./tsconfig.build.json",
         "build:cjs": "tsc --project ./tsconfig.build.json",
+        "push:update": "yarn clean && yarn build:esm && yarn publish",
```

Comment:
nit: `push:update` is un-reversible. Please do not test it locally if you don't have an intent to publish